### ISSUE TITLE
Breaking changes for engine strict in version 1.3.1

### ISSUE THREAD

**Issue by mtti-g** [shelfio/jest-mongodb#285]
For projects that use engine strict, the version `1.3.1` of `@shelf/jest-mongodb` is a breaking change.

Steps:
```bash
$ node -v
v12.19.0

$ npm -v
6.14.8

$ echo engine-strict=true >> .npmrc
$ npm i -s @shelf/jest-mongodb
```

Error:
```bash
npm ERR! code ENOTSUP
npm ERR! notsup Unsupported engine for mongodb-memory-server@7.0.0: wanted: {"node":">=12.22.0"} (current
: {"node":"12.19.0","npm":"6.14.8"})
npm ERR! notsup Not compatible with your version of node/npm: mongodb-memory-server@7.0.0
npm ERR! notsup Not compatible with your version of node/npm: mongodb-memory-server@7.0.0
npm ERR! notsup Required: {"node":">=12.22.0"}
npm ERR! notsup Actual:   {"npm":"6.14.8","node":"12.19.0"}
```

Previous versions did not have this requirement.

**Comment by vladgolubev** [comment]
@mtti-g  This is the problem with `mongodb-memory-server`
![CleanShot 2021-07-07 at 12 11 17@2x](https://user-images.githubusercontent.com/3817380/124733013-74684180-df1c-11eb-8015-c25856c93108.png)


`@shelf/jest-mongodb` requires Node.js 8 or higher

https://github.com/shelfio/jest-mongodb/blob/master/package.json#L62-L64

**Comment by vladgolubev** [comment]
Here is the commit that introduced this change https://github.com/nodkz/mongodb-memory-server/commit/7d6d018a6fd3357a2ed82f663fd5218143ec0a2a#diff-e4fb67d7357160e9cfde80dde338ba26430ab721f1721f9ce3df3aa495e8ca25L12-L15

**Comment by hasezoey** [comment]
@vladgolubev this is an problem with this package here (`jest-mongodb`) because using an dependency that has an higher nodejs requirement, means the package that uses it, will also require that nodejs version (at least)

so your ways are:
- either revert to an version that does support NodeJS v8
- release an major version increasing the required NodeJS version

it is explicitly stated that `mongodb-memory-server` version 7.0.0 and up require NodeJS `>=12.22.0` (in changelog and migration guide)

**Comment by vladgolubev** [comment]
@hasezoey thanks for the detailed explanation 👍🏻 

I'm going to revert to a compatible version in 1.x and publish version 2.x that requires node 12+

**Comment by vladgolubev** [comment]
@hasezoey @mtti-g 

I've just published 1.3.2 (with Node 8 support) and 2.0.0 (with Node 12+ support)

https://github.com/shelfio/jest-mongodb/releases

Let me know does it work for you!

**Comment by mtti-g** [comment]
Thanks for looking into this! This resolves my issue. :confetti_ball:
